supportedBackends/sfs/src/main/scala/cromwell/backend/sfs/SharedFileSystem.scala

```diff
@@ -188,7 +188,7 @@ trait SharedFileSystem extends PathFactory {
       val dest = if (inputsRoot.isParentOf(localInputPath)) localInputPath
       else {
         // Concatenate call directory with absolute input path
-        DefaultPathBuilder.get(inputsRoot.pathAsString, localInputPath.pathAsString)
+        DefaultPathBuilder.get(inputsRoot.pathAsString, localInputPath.parent.pathAsString.hashCode.toString + "/" + localInputPath.name)
```

Comment:
`DefaultPathBuilder.get(inputsRoot.pathAsString, localInputPath.parent.pathAsString.hashCode.toString, localInputPath.name)`

Reply:
I imagine the answer is yes but if 2 strings have the same value do they necessarily have the same `hashCode` ?
We want it to be the case so that files that were originally in the same directory always end up in the same "shorten" directory.

Reply:
yes, strings have the same hashcode when they are effectively the same value (even if a different instance)